sympy/physics/units/definitions/unit_definitions.py

```diff
@@ -329,8 +329,23 @@
 # Other convenient units and magnitudes
 
 ly = lightyear = lightyears = Quantity("lightyear", abbrev="ly")
+ly.set_global_relative_scale_factor(9_460_730_472_580.8, kilometer)
```

Comment:
I wouldn't use floating-point numbers in SymPy, unless really necessary, better:
`ly.set_global_relative_scale_factor(9_460_730_472_580_800, meter)`

Reply:
Need to remove `SI.set_quantity_scale_factor(lightyear, speed_of_light*julian_year)` from SI definitions (as it should be valid in all unit systems).